index.bs

```diff
@@ -131,27 +144,72 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
-</li>
-<li>The track handlers <code>vide</code> or <code>pict</code> MAY be used.
-</ul>
-
-<h2 id="alpha-images">Alpha Image Items and Sequences</h2>
-
+<p>
+    An <dfn>AV1 Image Sequence</dfn> is defined as a set of AV1 [=Temporal Units=] stored in an [=AV1 track=] as defined in [[!AV1-ISOBMFF]] with the following constraints:
+
+    - The [=AV1 Image Item=] should have its <code>[=still_picture=]</code> flag set to 1.
+    - The [=AV1 Image Item=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+    - The track handler for an [=AV1 Image Sequence=] shall be <code>pict</code>.
+
+    <h2 id="alpha-images">Alpha Image Items and Sequences</h2>
 <p>An <dfn>AV1 Alpha Image Item</dfn> (respectively an <dfn>AV1 Alpha Image Sequence</dfn>) is an [=AV1 Image Item=] (respectively [=AV1 Image Sequence=]</dfn>) with the following additional constraints:
-<ul>
-<li>The <code>[=mono_chrome=]</code> field in the [=Sequence Header OBU=] SHALL be set to 1.</li>
-</ul>
+    - The <code>[=mono_chrome=]</code> field in the [=Sequence Header OBU=] SHALL be set to 1.
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) SHALL be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-<h2 id="AVIF-baseline-profile">MIAF AV1 Baseline Profile</h2>
+  <h2>Brands and file extensions</h2>
+
+<h3>AVIF image and image collection brand</h3>
+Files conformant with the brand-independent restrictions in this document (sections [[#image-item-and-properties]] and [[#alpha-images]]) shall include the brand
+<dfn value="" export="" for="AVIF Image brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
+<p>Files should also carry a compatible brand to identify the AVIF profile (see section [[#profiles]]), if any, with which the file complies.</p>
+<p>If 'avif' is specified in the major_brand field of the FileTypeBox, the file extension should be ".avif".
+The MIME Content Type for files with the ".avif" file extension shall be "image/avif".</p>
+
+<h3>AVIF image sequence brand</h3>
+Files conformant with the brand-independent restrictions in this document (sections [[#image-item-and-properties]], [[#image-sequences]] and [[#alpha-images]]) shall include the brand
+<dfn value="" export="" for="AVIF Image Sequence brand">avis</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
+<p>Files should also carry a compatible brand to identify the AVIF profile (see section [[#profiles]]), if any, with which the file complies.</p>
+<p>If 'avis' is specified in the major_brand field of the FileTypeBox, the file extension should be ".avifs".
+The MIME Content Type for files with the ".avifs" file extension shall be "image/avif-sequence".</p>
```

Comment:
Same as above, should have an additional section for definition of the MIME type

Reply:
Done, but we can add the new section later.